### PR TITLE
document SHARD_ID

### PR DESCRIPTION
**Pull request by kvahed**

### CODE REVIEW

**Review by santanu82021 — COMMENTED** [review]
Looks good to me.

**Comment by joerg84** [comment]
Upstream PRs: 
devel https://github.com/arangodb/arangodb/pull/15428
3.9 https://github.com/arangodb/arangodb/pull/15430
3.8 https://github.com/arangodb/arangodb/pull/15431

**Review by KVS85 — CHANGES_REQUESTED** [review]
Need live examples.

**Review by KVS85 — APPROVED** [review]
LGTM.

**Review by ansoboleva — APPROVED** [review]
reviewed

File: 3.10/aql/functions-miscellaneous.md [hunk 1]
```diff
@@ -199,6 +199,29 @@ DECODE_REV( "_YU0HOEG---" )
 // { "date" : "2019-03-11T16:15:05.314Z", "count" : 0 }
 ```
 
+### SHARD_ID()
```
Comment (joerg84) [inline 3.10/aql/functions-miscellaneous.md:202]: Should this be mentioned in the changelog/what is new section as well?

**Review by joerg84 — APPROVED** [review]

**Pull request merged**